src/main/io/bf_osd_symbols.h

```diff
@@ -33,11 +33,16 @@
 #define BF_SYM_PITCH                   0x15
 #define BF_SYM_TEMPERATURE             0x7A
 
+// Glide symbols
+#define BF_SYM_GLIDESLOPE              0x05 
```

Comment:
BF_ defines should reflect the Betaflight font. Do not add things that don't exist in Betaflight.